### ISSUE TITLE
Benchmark results in numbers?

### ISSUE THREAD

**Issue by liutianlin0121** [oterobaguer/dip-ct-benchmark#1]
Hi! 

Thanks for this awesome resource. I am currently preparing a manuscript and it will be great if I can cite the numerical results in the Figure 9 of your DIP paper https://arxiv.org/pdf/2003.04989.pdf, where you plotted the test PSNR and SSIM results evaluated on the Ellipses and LoDoPaB dataset. But since these results are shown in the figure, I am not sure about their exact values. Would it be possible to have these PSNR and SSIM results in numbers? Also, it would be great to have the numbers on the validation set, too, if they are available. 

The reason I request these numbers is that, even with pre-trained models, it takes quite a lot of time to evaluate these models on the datasets, especially for DIP and TV because they are single-image based. Also, the pre-trained model in the dival package (https://github.com/jleuschn/dival) are all trained on 100% training data, and it will be great to have numbers of other percentages like those in Figure 9. Thanks for your consideration in advance.

Cheers,
Tianlin

**Comment by oterobaguer** [comment]
Hello Tianlin,

Thank you for your email. In the journal manuscript you can find the
numbers in the appendix.

https://iopscience.iop.org/article/10.1088/1361-6420/aba415

Please let me know if that helps.

Kind regards,
Daniel

**Comment by liutianlin0121** [comment]
Thanks so much!!
